pkgs/development/libraries/science/chemistry/tblite/python.nix

```diff
@@ -3,24 +3,52 @@
 , ninja
 , pkg-config
 , tblite
+, numpy
+, simple-dftd3
 , cffi
+, gfortran
+, blas
+, lapack
+, mctc-lib
+, mstore
+, toml-f
+, multicharge
+, dftd4
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   inherit (tblite) pname version src meta;
 
-  nativeBuildInputs = [ meson ninja pkg-config ];
-
-  buildInputs = [ tblite ];
-
-  propagatedBuildInputs = [ cffi ];
+  nativeBuildInputs = [
+    tblite
+    meson
+    ninja
+    pkg-config
+    gfortran
+    mctc-lib
+  ];
+
+  buildInputs = [
+    tblite
+    simple-dftd3
+    blas
+    lapack
+    mctc-lib
+    mstore
+    toml-f
+    multicharge
+    dftd4
+  ];
+
+  propagatedBuildInputs = [ tblite simple-dftd3 cffi numpy ];
+  patches = [ ./0001-fix-multicharge-dep-needed-for-static-compilation.patch ];
```

Comment:
Could you add a comment what the purpose of this patch is?

Reply:
Done